src/transactions/TransactionFrame.cpp

```diff
@@ -250,7 +251,10 @@ TransactionFrame::commonValid(SignatureChecker& signatureChecker,
         }
     }
 
-    if (mEnvelope.tx.fee < getMinFee(lm))
+	auto whitelisted =
+        Whitelist::instance(app)->isWhitelisted(mEnvelope.signatures, getContentsHash());
+
+    if (mEnvelope.tx.fee < getMinFee(lm) && !whitelisted)
```

Comment:
some conditions in the file check for `!whitelisted` first and then the remaining conditionals. don't see any reason not to do that here as well as anywhere else mainly for consistency (and optimization)